app/Http/Controllers/OAuthController.php

```diff
@@ -36,7 +36,14 @@ public function callback(string $service): RedirectResponse|Redirector
         };
 
         $email =  $authUser->getEmail();
-        [$fname, $lname] = explode(" ", $authUser->getName() ?? '');
+        $name = $authUser->getName() ?? '';
+
+        // Check if name has space.  Avoid issue where username = "Real" name
+        if(str_contains(" ", $name)) {
+          [$fname, $lname] = explode(" ", $name );
+        } else {
+            [$fname, $lname] = [$name, ""];
+        }
```

Comment:
It looks like the indentation is messed up here

Reply:
A couple things ended up being wrong here.  Thanks for the review!